### ISSUE TITLE
Not seeing missing method warning in stub file for an inherited method

### ISSUE THREAD

**Issue by msridhar** [typetools/checker-framework#7069]
We were trying to add a stub file for `org.apache.log4j.Logger` with an entry:

```java
    @SideEffectFree
    public void warn(Object message, Throwable t);
```

It turns out `warn` is declared in `org.apache.log4j.Category` and inherited by `Logger`, and we need the stub to be for the `Category` class for this to work.

Based on [the manual](https://checkerframework.org/manual/#stub-troubleshooting) I would have expected to get a warning by default for this case (the stub was passed in via the `-Astubs` argument), but I did not.  But I think a warning would be appropriate for this case, as the written stub annotations are not being processed, even though the method exists.
